### PR TITLE
[RUIN-312], [RUIN-307] Update JSON converter and HTML strings to use the human readable ids

### PR DESCRIPTION
**Pull request by 18chowdhary**
# Description

Previously, the HTML strings used to generate HTML and PDF versions of the report were not using the new human-readable IDs but instead relying on the old randomly-generated 8-character IDs from the spreadsheet. Additionally, every component was submitting its data using other randomly-generated ids. For consistency purposes, every component has been updated to submit data using the question's human-readable id, and all of the JSON converters have been updated to reference these ids when looking for the corresponding answers to populate the export format. I manually updated all the HTML strings (the templates for the HTML and PDF exports) to use the human-readable IDs, but there are still some old ids because I cannot tell which questions these ids correspond to.

# Testing
1. Start a new report.
2. Add 2 vehicles and one non-motorist.
3. Populate each of the forms. Make sure to answer a number of questions. Try filling out one text field, one number button selector, one multi-button selector, one single select dropdown, and one multi-select dropdown in each of the forms.
4. Export the form to a desired format. (Try HTML or PDF).
5. Confirm that the export worked correctly.

### CODE REVIEW

**Comment by KristinAoki** [comment]
Issues that i found while filling out form
- Time autofill no longer appears to be working
- county doesn't register when state has been selected

**Comment by 18chowdhary** [comment]
> When filling out the different forms, none of the information was populated. Even the part where the photo used to appear does not show up. I did notice that there were a lot of accepting errors that occurred when I was filling out various parts of the forms but didn't break the app.

I addressed this issue, and the dependencies and auto-fills should be working now! (I can guarantee that the county drop-down and the time autofill are working, unsure about the rest).

File: src/reducers/RoadReducer.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ export default function roadReducer (state=initialState, action) {
             const { id, question, selection } = action.payload;
             let updatedState = state.data;
             let roadData = updatedState.find(road => road.id == id);
+            // console.log('question answered for road:', question)
```
Comment (KristinAoki) [inline src/reducers/RoadReducer.js:24]: delete these

**Review by KristinAoki — CHANGES_REQUESTED** [review]
The form is filled out. But when I look at the PDF preview only the road sections are present and the headers at the top of the form are no longer present. 

Also please delete all log statements that are commented out

**Review by KristinAoki — APPROVED** [review]

**Pull request merged**